### DISCUSSION TITLE
[Feature] show current value in alarm message

### DISCUSSION THREAD

**Discussion by zhangethan** [apache/skywalking#8996] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Now , alarm message looks like this:
Successful rate of endpoint {name} is lower than 75%
Could you please support current value? For example:
Successful rate of endpoint {name} is lower than 75%, it's {value}

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment, marked as answer]
The alarm running context bases on a time windows, there is no concept about `{value}`.

**Reply by Jack-Ye** [reply]
Time windows should have the sum output, and this is a really good feature, should be supported

**Reply by wu-sheng** [reply]
You have so many `should`, how about you should work on this? Working for the community and others?
